src/api/app/views/webui/users/token_triggers/show.html.haml

```diff
@@ -0,0 +1,34 @@
+- @pagetitle = 'Trigger Token'
+
+.card
+  .card-body
+    %h3.mb-3= @pagetitle
+
+    -# "as: :token" to have a consistent name across all various token classes
+    = form_for(@token, as: :token, url: token_trigger_path, method: :put, local: false) do |f|
+      .form-row
+        .col-12.col-md-10.col-lg-9
+          .form-group
+            = f.label(:id, 'Id:')
+            = @token.id
+          .form-group
+            = f.label(:type, 'Operation:')
+            = @token.class.token_name.capitalize
+          - token_package = @token.package
+          - if token_package.present?
+            = fields_for(:package, token_package) do |token_fields|
+              .form-group
+                = token_fields.label(:project, 'Project:')
+                = token_package.project
+              .form-group
+                = token_fields.label(:package, 'Package:')
+                = token_package.name
+          - else
+            .form-group.row
+              .col-sm
+                = render partial: 'webui/shared/autocomplete', locals: { html_id: 'project_name', label: 'Project:', required: true,
+                                                                         data: { source: autocomplete_projects_path(local: true) } }
+              .col-sm
+                = render partial: 'webui/shared/autocomplete', locals: { html_id: 'package_name', label: 'Package:', required: true,
```

Comment:
This seems broken if results is a single package. Just tried in the review app....

![Screenshot from 2021-09-09 13-54-32](https://user-images.githubusercontent.com/514785/132681240-7f70ef76-936e-478e-bb3a-de30c074ca38.png)

![Screenshot from 2021-09-09 13-54-55](https://user-images.githubusercontent.com/514785/132681297-f7ac326c-ff2a-4465-ac6c-4b95ec3b65af.png)

Reply:
alright, will have a look!